### ISSUE TITLE
User Feedback: Document always idles out after 15 minutes

### ISSUE THREAD

**Issue by Ezinnem** [CollaboraOnline/online#6919]
**Describe the bug**
if I leave a document alone for 15 minutes the document disconnects from the server and is never able to connect.  I have to close the window and open Nextcloud again and then select the file again.

**To Reproduce**
Steps to reproduce the behaviour:
1. Open  a file
2. Allow to idle for 15 minutes
The server disconnects and is not able to reconnect

**Expected behaviour**
The server does not idle out within 15 minutes

**Actual behavior**
The server disconnects and is nit able to reconnect

**Comment by timur-g** [comment]
That shoud be idle timeout. But I do not have that experience to have to close the window. 
It would be good ot know Online version, configuration, browser, if file is opened directly or via recent files in Nextcloud.

**Comment by Darshan-upadhyay1110** [comment]
@Ezinnem can you confirm this bug. i can not reproduce this. Please close it if no more timeout issue happening

**Comment by rpear** [comment]
Yes, I would say close it, also Ezinnem hasn't confirmed the bug.
